### PR TITLE
Full website

### PR DESCRIPTION
**Pull request by Amedzro-Elikplim**
- Add home and quote page
- Style pages
- Add a navbar
- Create routes to pages using the react-router library

### CODE REVIEW

File: src/components/Navbar/Nav.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import '../../styles/Nav.css';
+import React from 'react';
+import { Link } from 'react-router-dom';
+
```
Comment (alexander16108) [inline src/components/Navbar/Nav.js:4]: - [ ] 
I would strongly recommend you fix the live demo of the project, I suggest you use netlify to deploy your projects

**Review by alexander16108 — CHANGES_REQUESTED** [review]
Hi @Amedzro-Elikplim, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: src/components/Navbar/Nav.js (hunk 1 above)
Comment (Amedzro-Elikplim) [reply src/components/Navbar/Nav.js:4]: @alexander16108  thanks for the feedback.
The site is live on Netlify: https://math-magicians-microverse.netlify.app/

**Review by vic778 — APPROVED** [review]
Hi @Amedzro-Elikplim    , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Charlie Brown

## Highlights

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification

**Pull request merged**